packages/firebase_auth/ios/Classes/FirebaseAuthPlugin.m

```diff
@@ -170,13 +170,13 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
                       }];
   } else if ([@"reauthenticateWithCredential" isEqualToString:call.method]) {
     [[self getAuth:call.arguments].currentUser
-        reauthenticateWithCredential:[self getCredential:call.arguments]
+        reauthenticateAndRetrieveDataWithCredential:[self getCredential:call.arguments]
```

Comment:
This diverges from the google3 change. I will make this change and make sure it is compatible and if not, I need to talk to firebase folks and see what they recommend. See history of FirebaseAuthPlugin.m in google3 if you are curious.

Reply:
It looks like in google3 `reauthenticateWithCredential` and `reauthenticateAndRetrieveDataWithCredential` do the same thing, so this change should work fine.